File: test/ocamlTests/CharTest/toDigitTest.ml
test "toDigit('0')" (fun () -> expect (Char.toDigit '0') |> toEqual (let open Eq in option int) Some(0)) ; 
test "toDigit('8')" (fun () -> expect (Char.toDigit '8') |> toEqual (let open Eq in option int) Some(8)) ; 
test "toDigit('a')" (fun () -> expect (Char.toDigit 'a') |> toEqual (let open Eq in option int) None) ; 

File: test/fsharpTests/CharTest/toDigitTest.fs
testCase "toDigit('0')" 
<| fun _ -> 
    let expected = Some(0)
    Expect.equal expected (Char.toDigit '0') "error"
testCase "toDigit('8')" 
<| fun _ -> 
    let expected = Some(8)
    Expect.equal expected (Char.toDigit '8') "error"
testCase "toDigit('a')" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Char.toDigit 'a') "error"
